python/src/aac/plugins/gen_protobuf/gen_protobuf_impl.py

```diff
@@ -362,6 +374,58 @@ def _convert_camel_case_to_snake_case(camel_case_str: str) -> str:
     return snake_case_str
 
 
+def _sanitize_message_to_camel_case(message_with_space: str) -> str:
```

Comment:
These function, and its siblings, might be more accurately named by removing the reference to message, something like `_convert_string_to_camel_case(string_to_convert: str) -> str`.